tools/WixBuild.props

```diff
@@ -105,7 +108,8 @@
     <PlatformSdkIncludePath Condition=" Exists('$(PlatformSdkIncludeRoot)\shared') ">$(PlatformSdkIncludePath);$(PlatformSdkIncludeRoot)\shared</PlatformSdkIncludePath>
     <PlatformSdkIncludePath Condition=" Exists('$(PlatformSdkIncludeRoot)\um') ">$(PlatformSdkIncludePath);$(PlatformSdkIncludeRoot)\um</PlatformSdkIncludePath>
     <PlatformSdkBinPath>$(PlatformSdkInstallPath)bin\</PlatformSdkBinPath>
-    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkBinPath)x86') ">$(PlatformSdkBinPath)x86\</PlatformSdkBinPath>
+    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkInstallPath)bin\x86') ">$(PlatformSdkInstallPath)bin\x86\</PlatformSdkBinPath>
+    <PlatformSdkBinPath Condition=" Exists('$(PlatformSdkInstallPath)bin\$(PlatformSdkBuildVersion)\x86') ">$(PlatformSdkInstallPath)bin\$(PlatformSdkBuildVersion)\x86\</PlatformSdkBinPath>
```

Comment:
On my machine, this is resolving to `C:\Program Files (x86)\Windows Kits\10\bin\10.0.14393.0\x86`. However, there is only genxbf.dll and the folder AppAnalysis inside of there so the build fails to find rc.exe.  The folder `C:\Program Files (x86)\Windows Kits\10\Include\10.0.14393.0\um` seems fine. I have the latest VS2015 on this machine.